### DISCUSSION TITLE
open new page for serach

### DISCUSSION THREAD

**Discussion by ccczhouxin** [squidfunk/mkdocs-material#4487] [Q&A] [answered]
How can i use search to a new page,not in this page,and when the input has nothing,it can not blank,can keep result in the page
![image](https://user-images.githubusercontent.com/43168824/194988609-ea831be5-c145-4c9b-9003-0b9dce22201c.png)
![image](https://user-images.githubusercontent.com/43168824/194988686-ceecd8d3-52c1-4c5e-a1b4-5394a74005f4.png)

**Comment by squidfunk** [comment, marked as answer]
Search results always show results for the query entered, so if nothing is entered, there are no results. Opening search results in a new page can be done by CMD + clicking or right clicking and "open in new tab". Automatically opening links in new tabs or windows is considered bad UX and thus will never be part of Material for MkDocs.

**Reply by ccczhouxin** [reply]
ok,Thank you
